maintainers/maintainer-list.nix

```diff
@@ -14171,4 +14171,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  rgrinberg = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.